lemmy.yml

```diff
@@ -166,9 +166,6 @@
             owner: "1000"
             group: "1000"
 
-      vars:
```

Comment:
i'm pretty sure `host_vars` are mapped against the host in the `inventory/hosts` file. As such, in the current playbooks we never import the whole file. We use `host_vars` but not how it was originally intended via ansible. 

Pretty sure we still need to actually import the variable file `inventory/host_vars/{{ domain }}/vars.yml` 

Because otherwise we make assumptions that the users `hosts` file exactly matches our format (see other comment)

Reply:
I am not following your comment. Assuming everyone follows the semantics of the README/docs, Ansible will auto-load the `vars.yml` in my example. You do not need to explicitly import it.

Reply:
> You do not need to import it explicitly

If your testing what does your host file look like. Does your "server" match your `domain=<server>`

Testing:  
- inventory/hosts
```
[lemmy]
my-server-name-which-doesnt-match-domain  ansible_user=root domain=example.com  letsencrypt_contact_email=your@email.com  lemmy_base_dir=/srv/lemmy
```

- inventory/host_vars/example.com/vars.yml
```
postgres_env_vars:
  - POSTGRES_USER: lemmy
```

- test.yml
```
--
- hosts: lemmy

  tasks:
  - name: Display all variables/facts known for a host
    ansible.builtin.debug:
      msg:
        - "Provisioning based on postgres_env_vars which is: {{ lookup('ansible.builtin.env', 'postgres_env_vars') }}"
```

result:
```
ok: [my-server-name-which-doesnt-match-domain] => {
    "msg": [
        "Provisioning based on pictrs_vars which is: "
    ]
}
```

----
changing: inventory/hosts so that `example.com` is now the "host"
```
[lemmy]
example.com ansible_user=root domain=example.com  letsencrypt_contact_email=your@email.com  lemmy_base_dir=/srv/lemmy
```

Results in ansible actually merging the host_vars. That is why they are called host_vars. As they are specific for the host.

Proof:

```
PLAY [lemmy] ********************************************************************************************************
ERROR! failed to combine variables, expected dicts but got a 'dict' and a 'AnsibleUnicode':
ansible error continues because we are attempting to print a dictionary object
```